src/constants/tokens.json

```diff
@@ -103,7 +103,7 @@
     "name": "SportX"
   },
   "FRAX": {
-    "address": "0x104592a158490a9228070E0A8e5343B499e125D0",
+    "address": "0x45c32fA6DF82ead1e2EF74d17b76547EDdFaFF89",
     "decimals": 18,
```

Comment:
What are we going to do with the old (pos) one shouldn't we have both or no?  Also don't we need to update the FRAX token logo since this is a new address

Reply:
Already added new token logo for this token.